examples/sequent_calculus/connective_rules/05b_conjunction_introduction_in_the_antecedent_01.n3s

```diff
@@ -0,0 +1,55 @@
+@prefix log: <http://www.w3.org/2000/10/swap/log#>.
+@prefix : <http://example.org/ns#>.
+
+# Conjunction Introduction in the Antecedent
+# Γ ϕ χ-> Γ (ϕ ∧ ψ) χ
```

Comment:
Isn't the condition that Γ ⊢ ψ additionally required here?

Reply:
It's not, because it is conjunction now; 
If A implies C, then (A AND B) implies C as well.

Reply:
You're right, I got it, thanks for the explanation! 😊